policy-service/README.md

```diff
@@ -20,8 +20,22 @@ limitations under the License.
 
 # Policy Service
 
-The responsibility of the Policy Service is to provide the set of rules (filters or transformations) that need to be
-applied to each resource that has been requested, based the User and Context.
+The responsibility of the Policy Service is to provide the "set of rules" (filters or transformations) that need to be applied to each resource that has been requested, based the User and Context.
+The "set of rules" are split into 2 groups:
+1. Resource Level Rules (If no resource level rules are set on a resource then access will not be granted to any users)
+1. Record Level Rules
+
+A resource will need to have at least 1 resource level and 1 record level rule associated with it, this can be as simple as a pass through rule that does nothing.
+If either of these are not associated with the resource then the system will assume that nobody has access to it.
+
+#### Resource Level Rules
+These rules define whether a `User` is allowed to access a resource.
+Therefore, the rules can only be based on the metadata about that resource coming from thee resource service, the attributes about the user coming from the user service and the query context that is passed in from the user making the query and enriched 
+by the palisade service.
```

Comment:
```suggestion
```

Reply:
I think this was done by IntelliJ messing with my layout. I've reverted this silly mistake

Reply:
and again